actions/parse-push/package.json

```
@@ -20,12 +20,13 @@
   "license": "AGPL-3.0",
```

This one is deprecated. Should be 

```suggestion
  "license": "AGPL-3.0-or-newer",
```

Or

```suggestion
  "license": "AGPL-3.0-only",
```

---

Ah, I didn't know that. GitHub still returns `AGPL-3.0` as spdx_id for repos with AGPL-3.0 🤷 

Let's set it to `AGPL-3.0-only` then since I don't know what the Exercism standard would be.

(Not gonna merge the suggestion myself right now to prevent issues since you're working on the branch)

---

Yeah, it's still _valid_, just deprecated 🥇